sim/global-variables.d.ts

```diff
@@ -37,7 +35,6 @@ declare global {
 	const Condition: DataType.Condition;
 	const RuleTable: DataType.RuleTable;
 	const Species: DataType.Species;
-	const Tools: DataType.Tools;
 	const TypeInfo: DataType.TypeInfo;
-	const toID: typeof DataType.Tools.getId;
+	const toID: typeof Dex.toID;
```

Comment:
Hm, I think `toID` should be moved to `server/`, and the rest of these should probably be moved into global-types.

Reply:
This can be deleted now, right?

```suggestion
```

Reply:
Moved it into server/'s globals, since it's still needed there.